src/components/Application/ApplicationStart.tsx

```diff
@@ -10,27 +11,62 @@ export interface ApplicationStartProps {
 }
 
 const ApplicationStart: React.FC<ApplicationStartProps> = (props) => {
-  const { template: type, sections, handleClick } = props
+  const { template, sections, handleClick } = props
+  const { name, code, startTitle, startMessage } = template
 
-  return type ? (
-    <Container text>
-      <Header as="h1" content={type ? type.description : 'Create application page'} />
-      {type && (
-        <Segment>
-          {sections && (
-            <Header as="h2" content={`Number of sections in template: ${sections.length}`} />
-          )}
-          <List>
-            {sections &&
-              sections.map((section) => (
-                <List.Item key={`list-item-${section.code}`} content={section.title} />
-              ))}
-          </List>
-          <Button content={type.name} onClick={handleClick} />
-        </Segment>
-      )}
-      {!type && <Label content="No Application" />}
-    </Container>
+  return template ? (
+    <Segment.Group style={{ backgroundColor: 'Gainsboro', display: 'flex' }}>
+      <Button as="href" icon="angle left" label={{ content: name, color: 'grey' }} />
+      <Header textAlign="center">{strings.TITLE_COMPANY_PLACEHOLDER}</Header>
+      <Segment
+        style={{
+          backgroundColor: 'white',
+          padding: 10,
+          margin: '0px 50px',
+          minHeight: 500,
+          flex: 1,
+        }}
+      >
+        <Header as="h2" textAlign="center">
+          {`${name} ${strings.TITLE_APPLICATION_FORM}`}
+          <Header.Subheader>{strings.TITLE_INTRODUCTION}</Header.Subheader>
+        </Header>
+        {template && (
+          <Segment basic>
+            <Header as="h5">{strings.SUBTITLE_APPLICATION_STEPS}</Header>
+            <Header as="h5">{strings.TITLE_STEPS.toUpperCase()}</Header>
+            <List divided relaxed>
+              {sections &&
+                sections.map((section) => (
+                  <List.Item key={`list-item-${section.code}`}>
+                    <List.Icon name="circle outline" />
+                    <List.Content>{section.title}</List.Content>
+                  </List.Item>
+                ))}
+            </List>
+            <Divider />
+            {startTitle && (
+              <Message info>
+                <Message.Header>
+                  <Icon name="info circle" />
+                  {startTitle}
+                </Message.Header>
+                {startMessage && (
+                  <Message.List>
+                    {startMessage.split(/\r?\n/).map((str, index) => (
+                      <Message.Item key={`start-item-${index}`}>{str}</Message.Item>
+                    ))}
+                  </Message.List>
```

Comment:
I don't think we should convert new lines into lists like this. If the template wants to define a list it should use a markdown list. For now, I'd suggest keeping it as simple as possible and just displaying the message as-is, then we can improve the layout when we add markdown interpretation. (See also my back-end comments regarding title and message seperation -- i.e. I don't think we need it)

Reply:
No problem. Now will be using markdown instead. Updating...

Reply:
Done: [11371fd](https://github.com/openmsupply/application-manager-web-app/pull/233/commits/11371fd5acbe75192639c4141832e7a421eb6c86)